README.md

```diff
@@ -180,3 +180,7 @@ category: "design"
 ├── README.md
 └── science.md
 ```
+
+
```

Comment:
The text does not comply to the set style.

*Origin: MarkdownBear, Section: `markdown`.*

The issue can be fixed by applying the following patch:

```diff
--- a/README.md
+++ b/README.md
@@ -181,6 +181,4 @@
 └── science.md
 ```
 
-
-
 <br> # # # Action Time # # # </br>      		Hello!
```